packages/atomic/src/components/result-template-components-v1/atomic-result-multi-text/atomic-result-multi-text.tsx

```diff
@@ -0,0 +1,164 @@
+import {Component, Prop, h, State, VNode} from '@stencil/core';
+import {
+  BreadcrumbManager,
+  buildBreadcrumbManager,
+  Result,
+  ResultTemplatesHelpers,
+} from '@coveo/headless';
+import {ResultContext} from '../../result-template-components/result-template-decorators';
+import {getFieldValueCaption} from '../../../utils/field-utils';
+import {
+  Bindings,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../utils/initialization-utils';
+import {titleToKebab} from '../../../utils/utils';
+
+const listItemClasses = 'inline-block';
+
+/**
+ * The `atomic-result-multi-text` component renders the values of a multi-value string result field.
+ * @part result-multi-text-separator - The separator between two field values.
+ * @part result-multi-text-value - A field value.
+ * @slot result-multi-text-value-* - Allows the personalization of a particular field value based on its indexed name.
+ */
+@Component({
+  tag: 'atomic-result-multi-text-v1',
+  styleUrl: 'atomic-result-multi-text.pcss',
+  shadow: true,
+})
+export class AtomicResultMultiText implements InitializableComponent {
+  public breadcrumbManager!: BreadcrumbManager;
+
+  @InitializeBindings() public bindings!: Bindings;
+  @ResultContext() private result!: Result;
+
+  @State() public error!: Error;
+
+  /**
+   * The result field which the component should use.
+   * This will look in the Result object first, and then in the Result.raw object for the fields.
+   * It is important to include the necessary field in the ResultList component.
+   */
+  @Prop() public field!: string;
+
+  /**
+   * If present, will attempt to split the field into multiple values using wherever this character is encountered.
+   */
+  @Prop() public delimitingCharacter?: string;
+
+  /**
+   * The maximum number of field values to display.
+   * If there are more values than the maximum that was set here, the last value displayed will be "# more...".
+   */
+  @Prop() public maximumValues = 3;
```

Comment:
Examples of what is displayed in the facet:
* maximumValues=3 with `English, French, Spanish`: "English, French, Spanish"
* maximumValues=4 with `English, French, Spanish`: "English, French, Spanish"
* maximumValues=3 with `English, French, Spanish, German`: "English, French, 2 more..."
* maximumValues=3 with `English, French, Spanish` and the "Spanish" value selected in a facet: "Spanish, English, French"

Reply:
No, since Johnny advised to skip "1 more" to avoid user frustration.

Reply:
Hmm, it feels unintuitive to expect a user to know about this off-by-one behavior.
If someone wanted that UX, wouldn't they set `maximumValues` to `2`?

Looking at the jsdoc too: `The maximum number of field values to display.`,
would `maximumValuesToDisplay` or `maxValuesToDisplay` be clearer names for this option?

Reply:
The opposite feels unintuitive too, it would be weird to set `maximumValues` to `2` and for 3 field values to show up.

I'm guessing the reason somebody would change `maximumValues` would be to allocate a certain amount of space, and the current behaviour seems more relevant in this regard.

I like the prop name you're suggesting, I'll use it.

Reply:
If `maximumValues` is set to 2, why would 3 values show up? I would expect only 2

Reply:
Indeed, I don't understand why you say that it's not intuitive.

I my mind, without reading any documentation, I would assume that setting `maximumValue=x` means that`I see X value, the rest is truncated`.

Reply:
> If `maximumValues` is set to 2, why would 3 values show up? I would expect only 2

> Indeed, I don't understand why you say that it's not intuitive.
> 
> I my mind, without reading any documentation, I would assume that setting `maximumValue=x` means that `I see maximum X value, anything above that is truncated.`

When some values are truncated, we want to show a "_n_ more..." label. However, we cannot show `1 more...` for UX reasons. Either there is 0 more, in which case we don't show a message, or either there is 2+ more. I will reiterate the current behaviour in a different format, because I think we have a misunderstanding.

|                     | maximumValues=3                | maximumValues=4                |
| ------------------- | ------------------------------ | ------------------------------ |
| 2 values from index | display 2 values               | display 2 values               |
| 3 values from index | display 3 values               | display 3 values               |
| 4 values from index | display 2 values & "2 more..." | display 4 values               |
| 5 values from index | display 2 values & "3 more..." | display 3 values & "2 more..." |

Is this the behaviour you would expect? If not, what would you expect?

Reply:
Going with the UX recommendation, wouldn't `maximumValues=3` and 5 values from the index be:
`display 3 values & 2 more`?

Since it is displaying the configured max, while still keeping a minimum of `2 more`?

Reply:
Good point, I didn't think about that. I'll implement that.